discovery-provider/src/tasks/entity_manager/track.py

```diff
@@ -173,11 +169,24 @@ def populate_track_record_metadata(track_record, track_metadata, handle):
     track_record.isrc = track_metadata["isrc"]
     track_record.iswc = track_metadata["iswc"]
     track_record.track_segments = track_metadata["track_segments"]
-    track_record.is_unlisted = track_metadata["is_unlisted"]
     track_record.field_visibility = track_metadata["field_visibility"]
-
     track_record.is_premium = track_metadata["is_premium"]
     track_record.is_playlist_upload = track_metadata["is_playlist_upload"]
+
+    if track_metadata["cover_art_sizes"]:
+        track_record.cover_art = track_metadata["cover_art_sizes"]
```

Comment:
Might want to add some tests to the tracks entity manager integration tests (`test_track_entity_manager.py`), not a huge deal tho

Reply:
Good call, will add!